tests/install_testbed/install_neutron_node.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash -x
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+# Authors: Jose Ignacio Carretero Guarde and Chema
+
+. ./config_vars
+
+INSTANCE_TUNNELS_INTERFACE_IP_ADDRESS=`ip a| awk "/inet $TUNNEL_IPS/  {print gensub(/\/../,\"\",\"g\",\\$2)}"`
+
+
+### configure rp_filter and ipforwarding.
+grep ^net.ipv4.ip_forward /etc/sysctl.conf
```

Comment:
In this case is not a good idea. You are thinking in code, but this is not code, this is a transcription of a documented installation process that have several phases. Here, there is not any advantage including the content in the same file and it is harder of maintain when a new version is published and you have to compare the changes from the old and the new version.

There is another reason here. You are comparing two scripts that usually are invoked in different hosts (one is for the neutron node, the other for each compute node).

I've only refactored when the operation is clearly independent and the same in each task (e.g. to create the database or register and endpoint). This is conservative, but it is easy to maintain. Parts that may be common between two services now may be not common in the next version. There is not any advantage in breaking the original "transcription" of the installation process.